package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "JavaScript mode",
-    "uri": "gh:zedapp/javascript-mode",
-    "version": "0.2",
+    "uri": "gh:thekiteeatingtree/javascript-mode",
```

Comment:
Yea sorry, I changed it so I could test/use it.  I should have made a branch to submit the PR.